src/attributes/testing.md

```diff
@@ -81,6 +81,48 @@ fn mytest() {
     assert_eq!(1, 2, "values don't match");
 }
 ```
+## Different panic strategies
+
+The `cfg_panic` feature makes it possible to exercise different lines of code depending on the panic strategy. 
+The possible value is either `unwind` or `abort`.
```

Comment:
Should this say make clear that code compiled with `#[cfg(panic = "unwind")]` being true may still aborting on panic if any crate is compiled with `-Cpanic=abort`?

Reply:
This is a good thing to point out.  Note, I'm not as familiar with crates so i don't know too much about this but I'll take your word for it.

I updated the file, let me know what you think :)